src/codewhisperer/service/keyStrokeHandler.ts

```diff
@@ -193,6 +201,7 @@ export class KeyStrokeHandler {
                     RecommendationHandler.instance.isGenerateRecommendationInProgress = false
                 }
             } else if (isInlineCompletionEnabled()) {
+                if (InlineCompletionService.instance.isPaginationRunning()) return
```

Comment:
This if is not necessary because .getPaginatedRecommendation contains this if

Reply:
done, thank!